programs/dart-format.nix

```diff
@@ -12,7 +12,7 @@ in
 
   config = lib.mkIf cfg.enable {
     settings.formatter.dart-format = {
-      command = cfg.package;
+      command = lib.getExe' cfg.package "dart";
```

Comment:
https://github.com/NixOS/nixpkgs/pull/342479 will resolve the dart warning once it propagates through hydra.